src/vm/eetwain.cpp

```diff
@@ -4816,8 +4816,22 @@ bool EECodeManager::EnumGcRefs( PREGDISPLAY     pContext,
                         printf("%cP+%02XH]: ", frameType, +dspOffs);
                 }
 #endif
+
+                unsigned flags = CHECK_APP_DOMAIN;
+#ifndef WIN64EXCEPTIONS
```

Comment:
@BruceForstall I think we should keep this change here for the sake of !WIN64EXCEPTION(i.e. x86/Windows). Since we want to change x86/Linux only.